FinniversKit/Sources/Components/Feedback/FeedbackView.swift

```diff
@@ -359,8 +359,8 @@ private class ButtonView: UIView {
 private extension Button.Style {
     static func negativeButton() -> Button.Style {
         Button.Style.default.overrideStyle(
-            borderWidth: 1.0,
-            borderColor: .backgroundPrimary
+            borderWidth: 2.0,
```

Comment:
should the borderWiidth change from 1 to 2?

Reply:
yes that was intentional.